Add enum arrays to consts file

parkerra opened the pull request:
## Add Enum Arrays to Consts File
The arrays for enums in the schema models for Dog and Log have been moved to the utils/consts file so that these constant string arrays can be used in multiple locations throughout the codebase, and every instance of these arrays can be updated at the same time if there are any changes.

### Checklist
- [x] Requirements have been implemented
- [x] Acceptance criteria is met
- [x] Database schema docs have been updated or are not necessary
- [x] Code follows design and style guidelines
- [x] Commits follow guidelines (concise, squashed, etc)
- [x] Relevant reviewers (EM) have been assigned to this PR

parkerra committed:
Add enum arrays to consts file

The arrays for enums in the schemas models for Dog and Log
have been moved to the utils/consts file so that these
constant string arrays can be used in multiple locations
throughout the codebase.

afazio1 commented on server/db/models/Log.js:
```diff
@@ -9,7 +10,7 @@ const LogSchema = new Schema({
+    enum: consts.topicArray,
     required: true,
   },
   tags: {
```
This is on me, I didn't add an enum property in the mongoose Log model for `tags`, but we would also like to create a consts array for `["auditory", "heartworm", "fleas/ticks"]`. 

Could you change this tags definition to:
```js
tags: {
    type: [String],
    enum: ["auditory", "heartworm", "fleas/ticks"]
  }
```

afazio1 commented on src/utils/consts.js:
```diff
@@ -10,6 +10,23 @@ const pages = {
 
 const consts = {
   baseUrl: process.env.BASE_URL ?? "http://localhost:3000",
+
```
Could you also add a `tagsArray` here with values `["auditory", "heartworm", "fleas/ticks"]`?

afazio1 reviewed (changes requested):
Great work! This will definitely help us when these values change!

I left a few comments about the `tags` property on the `Log` model. Again, I forgot to account for enums in the schema for `tags` and it would be great if you could add that in. Thanks, Parker!

parkerra committed:
Change array names and add tags array

The array names for genderArray and genderOtherArray have been
changed to genderPetArray and genderPersonArray to avoid
confusion. Also, a tagsArray has been added for Log tags
and the Log schema has been updated with the tagsArray as well.

afazio1 reviewed (approved):
Looks good! 👍
